pkg/semtech-loramac/Makefile.include

```diff
@@ -0,0 +1,9 @@
+INCLUDES += -I$(RIOTBASE)/pkg/semtech-loramac/include \
```

Comment:
Are all of these needed for *using* loramac? Everything internal to the module should go into Makefile.

Reply:
Yes, it seems to be the case. I tried different things but none worked (I'm not a build system expert though)

Reply:
This has nothing to do with the build system. If added this way, all these includes will be applied *globally* (to all modules). Please confirm that this is necessary, or maybe can be shrunk a little.